airbyte-integrations/connectors/source-partnerstack/source_partnerstack/manifest.yaml

```diff
@@ -4,73 +4,112 @@ definitions:
   selector:
     extractor:
       field_path: ["data", "items"]
-  requester:
-    url_base: "https://api.partnerstack.com/api/v2/"
+
+  schema_loader:
+    type: JsonFileSchemaLoader
+    file_path: "./source_partnerstack/schemas/{{ parameters.get('schema_name') or parameters['name'] }}.json"
+
+  base_requester:
+    url_base: "https://api.partnerstack.com/api/v2"
     http_method: "GET"
     authenticator:
       type: BasicHttpAuthenticator
       username: "{{ config['public_key'] }}"
       password: "{{ config['private_key'] }}"
-    request_parameters:
-      min_created: "{{ timestamp(config['start_date']) * 1000 }}"
-  partition_router:
-    request_cursor_field: "min_updated"
-    cursor_field: "updated_at"
-    class_name: source_partnerstack.components.PartnerstackSlicer
+
+  paginator:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: starting_after
+    page_size_option:
+      type: RequestOption
+      field_name: limit
+      inject_into: request_parameter
+    pagination_strategy:
+      type: CursorPagination
+      page_size: 250
+      cursor_value: "{{ last_records[-1]['key'] }}"
```

Comment:
@KimPlv please check the breaking change regarding the `last_records` macro in this [thread](https://airbytehq.slack.com/archives/C027KKE4BCZ/p1714420443899759).

Reply:
thanks, updated :white_check_mark: